index.html

```diff
@@ -26,12 +26,21 @@ <h1>Welcome</h1>
           <img src="./images/Union.svg" alt="Toggle button to display hidden nav items">
         </nav>
       </div>
+    <div class="desktop-nav">
+      <nav>
+        <ul>
+          <li>Portfolio</li>
+          <li>About</li>
+          <li>Contact</li>
+        </ul>
+      </nav>
     </div>
-  </header>
+  </div>
+  
   <section>
     <div class="headline-intro">
       <div class="header-content">
-        <h1 class="intro">Hey there. I'm Oyinlade <br>
+        <h1 class="intro">Hey there. I'm Oyinlade. <br>
           <span class="hello-span">I'm a software developer</span></h1>
         <p class="bio"> I can help you build a product,feature or website. Look through some of my work and
           experience! If you like what you see and have a project you need coded,
```

Comment:
- [ ] You don't have a full background image and the spacing is not there in your project so Kindly implement it. See the image below
### your design
![his](https://user-images.githubusercontent.com/81727988/171646661-1d9ff326-9c92-4850-bd70-2158929bff4d.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/171646700-ff902b91-6d2e-4bf9-810c-45d6ee58ab3b.png)

Reply:
- This issue is well resolved :heavy_check_mark: